config/settings.yaml.example

```diff
@@ -30,8 +30,8 @@
 :hsts_enabled: true
 
 # Ruby on Rails version (available: 6.0, 6.1)
```

Comment:
I'm wondering: does this PR make it incompatible with Rails 6.0? If so, this should only mention Rails 6.1. If it doesn't, then I'd prefer to first merge a PR that uses the new style (so the setting actually works well) and once packaging is in place we merge the 6.1 change. (My guess is that changing from a hash to an array is incompatible and thus you should explicitly choose one or the other.)

Reply:
I don't think it's backward compatible, I'll remove the `6.0`